internal/pkg/api/APIServer.go

```diff
@@ -36,17 +40,17 @@ func Start(config *confTypes.Config) {
 	apiRouter.HandleFunc("/add", apiService.ApiPOST).Methods("POST")
 	// TODO: Immplement
 	//Configuration specific routes
-	//configRouter := router.PathPrefix("/config").Subrouter()
+	//configRouter := router.PathPrefix("/configs").Subrouter()
 	//
 	//authRouter := router.PathPrefix("/apikey").Subrouter()
```

Comment:
Any reason to keep these codes commented rather be removed? is this work in the initial phase?

Reply:
This is already removed